packages/beacon-node/src/db/repositories/stateDiffArchive.ts

```diff
@@ -0,0 +1,54 @@
+import {ChainForkConfig} from "@lodestar/config";
+import {Db, Repository} from "@lodestar/db";
+import {RootHex, Slot, ssz} from "@lodestar/types";
+import {bytesToInt, toHex} from "@lodestar/utils";
+import {Bucket, getBucketNameByValue} from "../buckets.js";
+import {getRootIndexKey} from "./stateArchiveIndex.js";
+
+export class StateDiffArchiveRepository extends Repository<Slot, Uint8Array> {
```

Comment:
add comment to this class, noting that only `putBinary(), getBinary()` are used so no need a type?